crates/uv/tests/lock.rs

```diff
@@ -90,8 +90,8 @@ fn lock_wheel_registry() -> Result<()> {
         [[distribution]]
         name = "project"
         version = "0.1.0"
-        source = "editable+file://[TEMP_DIR]/"
-        sdist = { url = "file://[TEMP_DIR]/" }
+        source = "editable+"
+        sdist = { path = "" }
```

Comment:
This is also a little weird too. I'm not quite sure what to do otherwise here. You could skip serializing empty paths, but then you'd wind up with `sdist = {}`, which is also weird...